src/components/Table/readme.md

```diff
@@ -981,6 +981,145 @@ function TableExample(props) {
+const data = [
+    {
+        name: 'Leandro Torres',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '1234qwerty',
+    },
+    {
+        name: 'Jose Torres',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '1234asdfgh',
+    },
+    {
+        name: 'Reinier',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '1234zxcvbn',
+    },
+    {
+        name: 'Sara',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '5678qwerty',
+    },
+    {
+        name: 'Tahimi',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '5678asdfgh',
+    },
+    {
+        name: 'Leo',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '5678zxcvbn',
+    },
+    {
+        name: 'Tahimi Leon',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012qwerty',
+    },
+    {
+        name: 'Alejandro',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',  
+        id: '5678zdfgdf',
+    },
+    {
+        name: 'Carlos',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '3434fgfgdf',
+    },
+    {
+        name: 'Luis',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012asdfgh',
+    },
+];
+const Title = styled.h1.attrs(props => props.theme.rainbow)`        
+    font-family: 'Lato Light';
+    font-size: 25px;
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+    `;  
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+    `;
+
+function TableCustomAction(props) {
+    const [data, setData] = useState(props.data);
+
+    function handleOnDeleteElement(id) {
+            console.log(id)
+            const newData = data.filter(e => e.id!==id);
+            setData(newData)
+        }
+    const customActionStyles ={marginLeft: "auto"};
+    const StyledActionCell = styled.div`
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        align-items: center;
+        margin: auto;
+        box-sizing: border-box;
+    `;
+    const CustomAction = ({row})=>(
+            <StyledActionCell>
```

Comment:
I think that if we use a `width = {60}` in the column, we don't have to add a wrapper or styles to the button. Also in this way, we better take advantage of the Table space because we keep always small the column.